src/vde_backend/syncRawFluksoData.py

```diff
@@ -685,8 +686,36 @@ def sync(cassandra_session):
 	showProcessingTimes(begin, setup_time, timer) 
 
 
+def processArguments():
+	""" 
+	process arguments
+	argument : custom mode : choose a interval between 2 specific timings
+	"""
+	argparser = argparse.ArgumentParser(
+		description=__doc__,
+		formatter_class=argparse.RawDescriptionHelpFormatter
+	)
+
+	argparser.add_argument(
+		"--custom", type=str,
+		nargs="*",
+		help="start timing and end timing. Format : YYYY-MM-DD_HH:MM:SS+0200"
+	)
+
+	return argparser
+
+
 def main():
 
+	argparser = processArguments()
+	args = argparser.parse_args()
+
+	# Custom timings argument
+	custom_timings = {}
+	if args.custom:
+		custom_timings["start_ts"] = pd.Timestamp((args.custom[0]).replace('_', ' '), tz="CET")
```

Comment:
Please, use the ISO format for the time string, i.e. YYYY-mm-ddTHH:MM:SS. `T` is a separator understood by time libraries while `_` is not.